pkg/releaseutil/kind_sorter.go

```diff
@@ -99,46 +103,42 @@ var UninstallOrder KindSortOrder = []string{
 	"Namespace",
 }
 
-// sortByKind does an in-place sort of manifests by Kind.
+// sort manifests by kind.
 //
 // Results are sorted by 'ordering', keeping order of items with equal kind/priority
-func sortByKind(manifests []Manifest, ordering KindSortOrder) []Manifest {
-	ks := newKindSorter(manifests, ordering)
-	sort.Stable(ks)
-	return ks.manifests
-}
+func manifestsSortedByKind(manifests []Manifest, ordering KindSortOrder) []Manifest {
```

Comment:
The public function in `manifest_sorter.go` is `SortManifests`. This raises two questions for me...

1. Should the manifest sorting be in that file due to the name?
2. Should the function name be `sortManifestsByKind` to follow a similar naming pattern?

Reply:
done